Graph/TimeSeries/graphTimeSeriesTimeDepth.m

```diff
@@ -96,7 +101,7 @@
         'Layer',        'top', ...
         'Tag',          'axis2D');
 
-if all(all(colormap == rkbwr))
```

Comment:
Can you please explain this change?

Reply:
Ah ok, I think I got it. Same pb as in setTimeSerieColorbarContextMenu.m for version >= r2017a ?